torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -305,17 +406,37 @@ def _export_fx_node_to_onnxscript(
 
             fx_name_to_onnxscipt_value[node.name] = output
             return
-
-        if node.target == operator.getitem:
-            # __getitem__ on Tensor or Sequence of tensors. Not tuple.
-            exporter_key = "getitem"
+        if (
+            isinstance(node.target, types.BuiltinFunctionType)
+            and node.target in function_dispatcher._BUILTIN_TO_EXPORTER_KEY_TABLE
+        ):
+            # This function expects built-in functions to have name like mul, mul_1, ...
+            # symbolic fx.graph contains built-in functions to calculate python values.
+            exporter_key = function_dispatcher._BUILTIN_TO_EXPORTER_KEY_TABLE[
```

Comment:
Can you assert if all inputs (in `node.args`) are tensor, int, float, and symint? I don't want to dispatch a random object such as `symfloat` to `aten::add` at this stage.

Reply:
Why not allow SymFloat?

Reply:
I have seen SymFloat when I tried to get symbolic mode of GPT2. It was used with __pow__. so I added it (GPT2 has some other exporting problem though). But if you have a strong reason of disallowing SymFloat for now, please LMK, and I will remove them.